build/Linux/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 build_deb: 
     build: ./build/deb 
-    command: bash -c "dos2unix /deb/build.sh && /deb/build.sh"
+    command: bash -c "dos2unix /deb/build.sh && chmod 777 /deb/build.sh && /deb/build.sh"
```

Comment:
the `dpkg-deb` command that creates the debian package requires permission in the range `[???, 755]` so 777 does not work. This command works for me.

`bash -c "dos2unix /deb/build.sh && chmod -R 755 /deb && /deb/build.sh"`

Reply:
Ahh.  That was mostly a placeholder just to make sure everything had a permissions check.  I can will updated it based on your note, and we can double  check again when we review your Debian PR.